Makefile

```diff
@@ -4,6 +4,7 @@ clean:
 	dotnet clean
 
 install:
+	@dotnet --version || (echo "Dotnet is not installed, please install Dotnet CLI"; exit 1);
```

Comment:
I didn't test this on Windows, but it seems like it's the same: https://www.microsoft.com/net/core#windowscmd